example/ios/Podfile

```diff
@@ -0,0 +1,42 @@
+# Uncomment this line to define a global platform for your project
+platform :ios, '14.0'
```

Comment:
ios version here and in file above (AppFrameworkInfo.plist) is different, we can choose 14 tbh